### ISSUE TITLE
Searching from a symlinked project path sometimes shows duplicate results

### ISSUE THREAD

**Issue by BinaryMuse** [atom/find-and-replace#129]
I have the following directory structure (due to some corporate stuff, don't ask):

```
/
  dev_exclusions
    src
      desk
  Users
    btilley
      src -> /dev_exclusions/src
```

I always open Atom for this project from `~/src/desk`.

Sometimes (not 100% of the time), when doing a Project Find, I get duplicate results--one result for the relative project path, and one for the "actual" path in `/dev_exclusions`. Here's an example:

![Example](https://www.evernote.com/shard/s23/sh/3afc7104-9943-4373-ab7b-79d8f089eb20/2e1106792fcd5387fa4f191e9b6889dc/deep/0/i.png)

(The image isn't showing through the GH proxy, so here is the URL: https://www.evernote.com/shard/s23/sh/3afc7104-9943-4373-ab7b-79d8f089eb20/2e1106792fcd5387fa4f191e9b6889dc/deep/0/i.png)

Interestingly enough, when I do observe this, there is only *one* duplicate result from `/dev_exclusions`, listed after the results from the relative project path, even if there are multiple results from the relative project path.

**Comment by izuzak** [comment]
Possibly related PR: https://github.com/atom/scandal/pull/21

**Comment by benogle** [comment]
This has been fixed by atom/scandal#21
